helm-service/chart/templates/deployment.yaml

```diff
@@ -78,7 +78,7 @@ spec:
           - name: LOG_LEVEL
             value: {{ .Values.logLevel | default "info" }}
           - name: USE_COMMITID
-            value: {{ .Values.commitIDenabled }}
+            value: "false"
```

Comment:
was this change intentional or should we keep the option of using `{{ .Values.commitIDenabled }}` here?